### DISCUSSION TITLE
Kenapa KYC Error 400?

### DISCUSSION THREAD

**Discussion by Yusufparker** [ivanwilliammd/satusehat-integration#132] [Q&A (Tanya Jawab)] [answered]
saya sudah melakukan encrypt message mmengikuti library yang ada di satusehat KYC, dan menghasilkan output encrypted message yang sesuai.
namun saat encrypted message itu saya request ke api generate-url mengalami ketidak sesuaian
{"metadata":{"code":"400","message":"Error: Bad Request"},"data":{"error":"failed to decrypt ciphertext: crypto/rsa: decryption error"}}

issue ini bersifat urgent

**Comment by ivanwilliammd** [comment, marked as answer]
Silakan coba kembali dengan environment production.
KYC hanya available di Production, bukan staging
